src/breadcrumb-group/full-dropdown.tsx

```diff
@@ -0,0 +1,88 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { Ref } from 'react';
+import clsx from 'clsx';
+
+import { CustomTriggerProps } from '../button-dropdown/interfaces';
+import InternalButtonDropdown from '../button-dropdown/internal';
+import InternalIcon from '../icon/internal';
+import { DATA_ATTR_FUNNEL_KEY, DATA_ATTR_RESOURCE_TYPE, FUNNEL_KEY_FUNNEL_NAME } from '../internal/analytics/selectors';
+import { CancelableEventHandler } from '../internal/events';
+import { BreadcrumbGroupProps } from './interfaces';
+
+import styles from './styles.css.js';
+
+interface FullCollapsedDropdownProps {
+  items: ReadonlyArray<BreadcrumbGroupProps.Item>;
+  onClick: CancelableEventHandler<{ id: string }>;
+  onFollow: CancelableEventHandler<{ id: string }>;
+}
+
+const metadataTypeAttribute = {
+  [DATA_ATTR_RESOURCE_TYPE]: 'true',
+};
+
+export const FullDropdown = ({ items, onClick, onFollow }: FullCollapsedDropdownProps) => (
+  <>
+    <InternalButtonDropdown
+      items={items.map((item, index) => {
+        const isCurrentPage = index === items.length - 1;
+        return {
+          id: index.toString(),
+          text: item.text,
+          href: isCurrentPage ? undefined : item.href,
+          disabled: isCurrentPage,
```

Comment:
Would it make sense to add the disabled attribute inside the button dropdown for items with isCurrentPage=true?

Reply:
not sure I understand what you mean here

Reply:
I mean that instead of passing down the `disabled: isCurrentPage` we can only pass `isCurrentPage`. In the button dropdown we can then set: `disabled: disabled || isCurrentPage`.

Reply:
got it, thanks! after further discussion, disabled has gone entirely in the most recent revision :D